X16 Reference - 04 - KERNAL.md

```diff
@@ -635,7 +635,7 @@ Jiffies are 1/60th seconds.
 
 Purpose: Get the date and time  
 Call address: $FF50  
-Communication registers: r0, r1, r2, r3  
+Communication registers: r0, r1, r2, r3L  
```

Comment:
Please double check the table of contents (pg 43 on the PDF generated from this PR). Says r3 there, not r3L.